code.html

```diff
@@ -5,6 +5,7 @@
     <link rel="stylesheet" type="text/css" href="style.css">
 </head>
 <body>
+    <h1>Welcome <span id="displayuser"></span>!</h1><br>
```

Comment:
![Screenshot 2024-01-13 at 8 57 10 AM](https://github.com/samardhk/mealplanner/assets/54699627/c6527d82-df59-47f9-9a88-0a1841cb7485)

Some styling stuff to be fixed here:
a) There's no space between the welcome header and the BMI Calculator header.
b) We want the page to flow vertically instead of horizontally, so lets put both the headers above the bmi calculator box. Use flex box for this!